classes/statistics/MetricsDAO.inc.php

```diff
@@ -94,9 +94,241 @@ protected function getAssocObjectInfo($submissionId, $contextId) {
 		$publishedArticleDao = DAORegistry::getDAO('PublishedArticleDAO');
 		$publishedArticle = $publishedArticleDao->getByArticleId($submissionId, $contextId, true);
 		if ($publishedArticle) {
-			$returnArray = array(ASSOC_TYPE_ISSUE, $publishedArticle->getIssueId());
+			$returnArray = [ASSOC_TYPE_ISSUE, $publishedArticle->getIssueId()];
 		}
 		return $returnArray;
 	}
-}
 
+	/**
+	 * Retrieves several statistics about submissions, with sums and averages
+	 * @param $contextId int The context id
+	 * @param $dateStart string The start date filter
+	 * @param $dateEnd string The end date filter
+	 * @param $sectionIds array A list of section IDs to filter the result set
+	 * @return array Returns a dictionary with statistics
+	 */
+	public function getSubmissionStatistics($contextId = null, $dateStart = null, $dateEnd = null, $sectionIds = null) {
+		// Access submission constants.
+		import('lib.pkp.classes.submission.Submission');
```

Comment:
I saw there's an autoloader in the application, but didn't check its "requirements". Should I replace these import calls?!

Reply:
The autoloader is in `lib/pkp/includes/functions.inc.php` in the `customAutoload` class. Off the top of my head, yes, you can remove `import` statements that would bring in classes from `lib/pkp/classes` and `classes`.